nebula-logger/tests/log-management/classes/LogMassDeleteExtension_Tests.cls

```diff
@@ -113,7 +113,7 @@ private class LogMassDeleteExtension_Tests {
 
             String deleteAccessError = 'You do not have access to delete logs records';
 
-            LogMassDeleteExtension extension = new LogMassDeleteExtension(controller);
+            new LogMassDeleteExtension(controller);
```

Comment:
This seems like this line could be removed altogether think?

Reply:
this one is necessary, otherwise the error doesn't get successfully loaded to the page

Reply:
Doh! That makes sense, ignore me!